cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -2465,6 +2506,49 @@
             "Key": "Name",
```

Comment:
For consistency shall we add the Application tag also to the compute nodes?

Reply:
Added 👍